Models/DTOs/UpdateFreelancerQualificationAndName.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace AonFreelancing.Models.DTOs
+{
+    public class UpdateFreelancerQualificationAndName
+    {
+
+        public string Name { get; set; }    
+
+
+        public string? qualificationName { get; set; }
```

Comment:
SpecializationName insted of qualificationName, and freelancer may have more than on specialization .